tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="NSPDK_candidateClust" name="NSPDK_candidateClusters" version="9.2">
 	<requirements>
-		<requirement type="package" version="0.1">graphclust-wrappers</requirement>
```

Comment:
Why is this so? This seems strange. But nevertheless, you can do this with `yield` in a macro.